src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -172,6 +186,8 @@
          (react/call :refresh-workspace this))))
    :component-did-update
    (fn [{:keys [prev-state state refs]}]
-     ;; when switching to the analysis tab, grab the ref to it so we can access the track picker button
-     (when (not= (:analysis-tab prev-state) (@refs ANALYSIS))
-       (swap! state assoc :analysis-tab (@refs ANALYSIS))))})
+     (if-let [analysis-tab (@refs ANALYSIS)]
```

Comment:
Honestly, I still don't like having this button here, but if we keep it, it's too tall. It should be the same height as the import attributes button in summary.